lib/states/co/aurora.R

```diff
@@ -59,13 +86,16 @@ clean <- function(d, helpers) {
       # https://app.asana.com/0/456927885748233/570989790365270
       citation_issued = TRUE,
       outcome = "citation",
-      subject_race = tr_race[
+      old_subject_race = tr_race[
         if_else(Ethnicity == "HISPANIC OR LATINO", "HISPANIC", Race)
       ],
-      subject_sex = tr_sex[sex]
+      new_subject_race = tr_race[`race/ethnicity`],
+      subject_race = coalesce(old_subject_race, new_subject_race),
+      subject_sex = tr_sex[sex],
+      # NOTE: Race and ethnicity are not differentiated from 2018 on.
+      raw_race = coalesce(Race, `race/ethnicity`)
```

Comment:
Could you explain the change/what's going on here more clearly? My understanding from the context here is that, pre-2018, the data had separate race and ethnicity variables, and from 2018 on there is just one ethnicity variable called `race/ethnicity`.

Reply:
Yup, that's exactly right.